### ISSUE TITLE
SA-MP Voice does not work on the host

### ISSUE THREAD

**Issue by ipsLuan-dev** [CyberMor/sampvoice#52]
Good!
I was doing an implementation of the sampvoice plugin on my server. I tested localhost with some friends and everything was fine! However, when uploading to a server host, it just doesn't work.

Detail: the plugin is connecting and recognizing everyone who has sampvoice installed normally!

I sincerely thank everyone for their help!

**Comment by bobitza** [comment]
Can you post the code ?

**Comment by icell76** [comment]
It does not work on the hosting, because the port for voice chat is closed(in order to exclude vulnerabilities).

**Comment by bobitza** [comment]
So what ports need to be enabled or to set one port but i see he take randomly port..

**Comment by ipsLuan-dev** [comment]
> Can you post the code ?

Merry Christmas!
The code is the cybermor standard.

**Comment by ipsLuan-dev** [comment]
> It does not work on the hosting, because the port for voice chat is closed(in order to exclude vulnerabilities).

Merry Christmas!
So, which devices should I release for VOIP to work without any interference?

**Comment by bluenxyz** [comment]
Yes, I have the same problem. On localhost (windows) voip works but on common host (linux) it doesn't work

**Comment by bluenxyz** [comment]
Complicated that there is no support from the developer of the plugin and no clarification about these problems. There are 43 problem topics with no answer.

**Comment by Venombrk** [comment]
You need to put your server ip address on network.cpp and compile the plugin on linux to work on server.

`sockaddr_in bindAddr {};

        bindAddr.sin_family = AF_INET;
        bindAddr.sin_addr.s_addr = inet_addr("193.203.39.155");
        bindAddr.sin_port = NULL;`
